### PR TITLE
Update kr.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**

### CODE REVIEW

File: channels/kr.m3u [hunk 1]
```diff
@@ -115,6 +115,8 @@ http://live.worktv.or.kr:1935/live/wowtvlive1.sdp/playlist.m3u8
 http://slive.ytn.co.kr/ytn/_definst_/ytn_stream_20190306/playlist.m3u8
 #EXTINF:-1 tvg-id="YTNDMB.kr" tvg-country="KR" tvg-language="Korean" tvg-logo="https://i.imgur.com/fdFQonN.jpg" group-title="",YTN DMB (270p)
 http://slive.ytn.co.kr:1935/dmb/ydlive_20140419_1/playlist.m3u8
+#EXTINF:-1 tvg-id="YTNSCIENCE.kr" tvg-country="KR" tvg-language="Korean" tvg-logo="https://i.imgur.com/FIzME5L.jpg" group-title="Science",YTN SCIENCE 
+https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UCZdBJIbJz0P9xyFipgOj1fA/live
```
Comment (freearhey) [inline channels/kr.m3u:119]: Source online
LIVE  대한민국 과학 채널 YTN사이언스📺 13-0-1 screenshot

but the link does not work
image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the link again.

File: channels/kr.m3u (hunk 1 above)
Comment (taodicakhia) [reply channels/kr.m3u:119]: This link is ok (query link always return 302 so check links is wrong)

Comment (Sphinxroot) [reply channels/kr.m3u:119]: well it work for me https://i.imgur.com/VaQZ8EZ.png could it be geo ?

Comment (jkemp814) [reply channels/kr.m3u:119]: Works on my end, Southwest, US

Comment (LaneSh4d0w) [reply channels/kr.m3u:119]: Works here, Romania

image

Comment (freearhey) [reply channels/kr.m3u:119]: now it works

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**